### DISCUSSION TITLE
Can't create Egress

### DISCUSSION THREAD

**Discussion by jim-huang-nj** [gravitl/netmaker#2231] [General]
I just installed netmaker.
Server: v0.18.6
When I created Egress Gateway, I use 0.0.0.0/0 .
I got error information:
Failed to create Egress Gateway:
IP Ranges Cannot Be Empty.

How to fix it?

**Comment by mattkasun** [comment]
From release notes for v0.18.6

Limitations

    Egress to 0.0.0.0/0 (internet gateways) is currently disabled. Will be re-implemented in a near-future release. If you use or need internet gateways, either stay with 0.17.1, or use the following list of ranges in place of 0.0.0.0/0:
    0.0.0.0/5,8.0.0.0/7,11.0.0.0/8,12.0.0.0/6,16.0.0.0/4,32.0.0.0/3,64.0.0.0/2,128.0.0.0/3,160.0.0.0/5,168.0.0.0/6,172.0.0.0/12,172.32.0.0/11,172.64.0.0/10,172.128.0.0/9,173.0.0.0/8,174.0.0.0/7,176.0.0.0/4,192.0.0.0/9,192.128.0.0/11,192.160.0.0/13,192.169.0.0/16,192.170.0.0/15,192.172.0.0/14,192.176.0.0/12,192.192.0.0/10,193.0.0.0/8,194.0.0.0/7,196.0.0.0/6,200.0.0.0/5,208.0.0.0/4
